python/ccf/proposal_generator.py

```diff
@@ -299,6 +301,30 @@ def update_recovery_shares(**kwargs):
 def set_recovery_threshold(threshold, **kwargs):
     return build_proposal("set_recovery_threshold", threshold, **kwargs)
 
+@cli_proposal
+def update_root_ca_cert(cert_name, cert_path, skip_checks=False, **kwargs):
+    cert_pem = open(cert_path).read()
+
+    if not skip_checks:
+        try:
+            cert = x509.load_pem_x509_certificate(cert_pem.encode(), crypto_backends.default_backend())
+        except:
```

Comment:
```suggestion
        except Exception:
```

Reply:
Any advantage with this?

Reply:
Yes, you only want to catch user-defined exceptions, otherwise you will swallow things like SyntaxError or KeyboardInterrupt, which you generally want to let bubble up.

Reply:
SyntaxError is actually derived from Exception, but yeah, KeyboardInterrupt is not. Makes sense.

Reply:
Yes, you're right about SyntaxError, my bad. I think the point still stands.